twoaxistracking/tests/test_shading.py

```diff
@@ -0,0 +1,125 @@
+from twoaxistracking import shading, layout
+from shapely import geometry
+import numpy as np
+import pytest
+
+
+@pytest.fixture
+def rectangular_geometry():
```

Comment:
Instead of duplicating fixtures in every test file, you can have a single definition in conftest.py: https://docs.pytest.org/en/6.2.x/fixture.html#conftest-py-sharing-fixtures-across-multiple-files